docs/cache/limits.mdx

```diff
@@ -0,0 +1,145 @@
+---
+sidebar_label: Quotas and Limits
+sidebar_position: 8
+title: Momento Cache service quotas
+pagination_next: null
+description: Examine the namespace and item-level quotas in place with Momento Cache and discover the default and tunable values for your workload based on your support plan.
+hide_title: true
+---
+
+# Namespace and item quotas in Momento Cache
+
+:::info
+Service quotas are directly affected by the support plan on your account. Both <ins>soft and hard limits</ins> are increasable for the Pro tier and Enterprise plans. If you would like to request an increase, please [contact sales](https://www.gomomento.com/contact-us).
+:::
+
+## Namespace and items
+
+<table className="customTable">
+  <thead>
+    <tr>
+      <th>Resource</th>
+      <th>Entity</th>
+      <th>Default</th>
+      <th>Basic support</th>
+      <th>Pro support</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr className="highlightRow">
+      <td colspan="5"><strong>Soft limits</strong></td>
+    </tr>
+    <tr>
+      <td>Operations per second</td>
+      <td>namespace</td>
+      <td>100</td>
+      <td>1,000</td>
+      <td>10,000</td>
+    </tr>
+    <tr>
+      <td>Hot storage</td>
+      <td>namespace</td>
+      <td>100 MB</td>
+      <td>1 GB</td>
+      <td>10 GB</td>
+    </tr>
+    <tr className="highlightRow">
+      <td colspan="5"><strong>Hard limits</strong></td>
+    </tr>
+    <tr>
+      <td>Time to live (TTL)</td>
+      <td>item</td>
+      <td colspan="3">24 hours</td>
+    </tr>
+    <tr>
+      <td>Item size</td>
+      <td>item</td>
+      <td colspan="3">5 MB</td>
+    </tr>
+    <tr>
+      <td>Collection element size</td>
+      <td><a href="../cache/develop/basics/datatypes#collection-data-types-cdts" target="_blank">element</a></td>
+      <td colspan="3">128 KB</td>
+    </tr>
+  </tbody>
+</table>
+
+## Operations
+
+Certain cache APIs can perform multiple operations in a single request. Multi-element operations are more efficient, so the cost of these APIs is discounted at a **2:1 ratio**. This means every two elements will count as one operation towards the limiter.
+
+:::tip
+To further reduce the number of operations charged against your account, look into setting the [read concern](./learn/how-it-works/read-concern) header to `Express`. This will reduce the charged operations count to 0.8x the default value, and will reduce latencies for frequently accessed keys.
+:::
+
+As an example, a `SetAddElements` request adding one or two elements costs *one operation*, but with three or four elements it costs *two operations*, and so on.
+
+<table class="customTable">
+  <thead>
+    <tr>
+      <th>API Name</th>
+      <th>Operations</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr class="highlightRow">
+      <td colspan="2"><strong>Single operations</strong></td>
```

Comment:
Do we need the single ones? Like we can say everything other than multi is single. I'd move the multi up the page either way as I believe that's the more interesting one.